### ISSUE TITLE
examples from https://github.com/sphinx-doc/sphinx/blob/master/tests/roots/test-ext-doctest/doctest.txt are failed for me

### ISSUE THREAD

**Issue by ArtyomKaltovich** [thisch/pytest-sphinx#24]
Hello.

I stuck with test with `testoutput` with `:hide:` directive. I run the code as follow:

    $ pytest --doctest-modules  --doctest-glob="*.rst"

Unfortunately the examples page are outdated.

Regards.

**Comment by ArtyomKaltovich** [comment]
The rst is following:

```
Testing the doctest extension
=============================

* options for doctest/testcode/testoutput blocks

  .. testcode::

     print('Output         text.')

  .. testoutput::
     :hide:
     :options: +NORMALIZE_WHITESPACE

     Output text.
```

**Comment by thisch** [comment]
Hi @ArtyomKaltovich. Sry for my late reply. 

Can you check if the latest release of pytest-sphinx has fixed the issue for you? In #17 I added support for ignoring the `:hide:` option.

**Comment by ArtyomKaltovich** [comment]
Hello @thisch.

I've found a workaround, but thank you for the fix. :)

I will close the issue.
